contents/forms/button-group/basic-button-group/metadata.json

```
@@ -1,7 +1,7 @@
 {
   "en": {
-    "title": "BasicButtonGroup",
-    "description": "A basic ButtonGroup with three buttons labeled Years, Months, and Days."
+    "title": "Basic Button Group",
+    "description": "A basic Button Group with three buttons labeled Years, Months, and Days."
```

This is an English sentence, so they should be in lower case.
```suggestion
    "description": "A basic button group with three buttons labeled Years, Months, and Days."
```